assets/sass/components/global/_googlesitekit-header.scss

```diff
@@ -115,6 +115,21 @@
 
 			.mdc-button--dropdown {
 				padding-right: $grid-gap-desktop;
+
+				@media (max-width: $bp-mobileOnly) {
+					padding-right: 8px;
+				}
+			}
+
+			@media (max-width: $bp-mobileOnly) {
```

Comment:
I would move this block to be just under `.googlesitekit-date-range-selector` in the file, i.e before the styles for `svg`. Once again, there is nothing wrong but it's easier to read and you know immediately those styles are for `.googlesitekit-date-range-selector`. Where you have it right now, i must scroll back to know for which selector those styles apply for.